src/languages/en.ts

```diff
@@ -3181,7 +3181,9 @@ export default {
         overLimitAttendee: ({formattedLimit}: ViolationsOverLimitParams) => `Amount over ${formattedLimit}/person limit`,
         perDayLimit: ({formattedLimit}: ViolationsPerDayLimitParams) => `Amount over daily ${formattedLimit}/person category limit`,
         receiptNotSmartScanned: 'Receipt not verified. Please confirm accuracy.',
-        receiptRequired: (params: ViolationsReceiptRequiredParams) => `Receipt required${params ? ` over ${params.formattedLimit}${params.category ? ' category limit' : ''}` : ''}`,
+        receiptRequired: ({formattedLimit, category}: ViolationsReceiptRequiredParams) =>
+            // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
```

Comment:
Can we write this as a function with if/elses to make it more readable and not need to suppress this?

Reply:
I see this was resolved, but the lint suppression is still there.

Reply:
This point was resolved .. I refactored the `receiptRequired` to be more readable instead of the one line function. let me know if there is another issue.

Reply:
Yes we should to remove `// eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing`

Reply:
Oh i see ..  but typescript still complains if i remove the suppression line: 
<img width="520" alt="image" src="https://github.com/Expensify/App/assets/59809993/191db78c-db92-4a9e-85f2-307b378022cb">

FYI here is [the discussion](https://github.com/Expensify/App/pull/44220#discussion_r1650187870) where we decided to use `||` instead of `??` 


please let me know what you think and how we should proceed.

cc @ahmedGaber93

Reply:
Perhaps you can explicitly check if they are undefined? Since it seems like they can either be strings or undefined.

Reply:
> Since it seems like they can either be strings or undefined.

If that, `??` will be good enough to check undefined values, we can use it and remove lint suppression. @abzokhattab

Reply:
Reverted to `??`